pkgs/development/tools/rust/cargo-generate/default.nix

```diff
@@ -2,19 +2,21 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "cargo-generate";
-  version = "0.5.3";
+  version = "0.11.0";
 
   src = fetchFromGitHub {
     owner = "ashleygwilliams";
     repo = "cargo-generate";
     rev = "v${version}";
-    sha256 = "sha256-RrDwq5VufMDsPlqRmBP3x2RUWU740L0L18noByO1IDY=";
+    sha256 = "sha256-082rFxC/p68X8g58I7Q7Of70ymq7VsLhkQpcqVx0u/A=";
   };
 
-  cargoSha256 = "sha256-/0pxEQFhovPRI4Knv5xq6+PHRuGN6+tF8CdK5X30LKI=";
+  cargoSha256 = "sha256-1yTH5FvalzBD13FXh/X1jmZhVyWU3thrjNSsrHUfBNE=";
 
   nativeBuildInputs = [ pkg-config ];
 
+  patches = [ ./0001-tests-integration-favorites-skip-github-lookup.patch ];
```

Comment:
Can we use --skip instead https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/blockchains/openethereum/default.nix#L41 ?

Reply:
Oooh, didn’t know about it, sorry =). Replaced with your version.